packages/hawtio/src/plugins/shared/workspace.ts

```diff
@@ -46,6 +54,68 @@ class Workspace {
     })
   }
 
+  private maybeUpdatePlugins(response: IResponse): void {
+    if (this.pluginUpdateCounter === null) {
+      this.pluginUpdateCounter = response.value as number | null
+      return
+    }
+    if (this.pluginUpdateCounter !== response.value) {
+      if (parseBoolean(localStorage['autoRefresh'])) {
```

Comment:
we shouldn't directly touch localStorage here. like `updateRate` this parameter should be managed by `jolokiaService`. lets create save/load methods on jolokiaService.

like `updateRate` this parameters should be also able to change at the Jolokia preferences pane.